### PR TITLE
[AppConfig] Update appconfig help messages

### PR DESCRIPTION
**Pull request by ChristineWanjau**
**Related command**

**Description**
This PR updates the app config help messages. It updates the string  "**App Configuration**" to "**App Configuration store**", updates "keyvalues" strings to "key-value" and adds  the actual argument name in the description for the auth-mode. 

**Testing Guide**

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AppConfig

File: src/azure-cli/azure/cli/command_modules/appconfig/_params.py [hunk 1]
```diff
@@ -111,26 +111,26 @@ def load_arguments(self, _):
...
+        c.argument('endpoint', help='If auth mode is "login", provide endpoint URL of the App Configuration store. The endpoint can be retrieved using "az appconfig show" command. You can configure the default endpoint using `az configure --defaults appconfig_endpoint=<endpoint>`', configured_default='appconfig_endpoint')
         c.argument('auth_mode', arg_type=get_enum_type(['login', 'key']), configured_default='appconfig_auth_mode', validator=validate_auth_mode,
                    help='This parameter can be used for indicating how a data operation is to be authorized. ' +
                    'If the auth mode is "key", provide connection string or store name and your account access keys will be retrieved for authorization. ' +
-                   'If the auth mode is "login", provide the store endpoint or store name and your "az login" credentials will be used for authorization. ' +
+                   'If the auth mode is "login", provide the store endpoint (--endpoint) or store name (--name) and your "az login" credentials will be used for authorization. ' +
```
Comment (albertofori) [inline src/azure-cli/azure/cli/command_modules/appconfig/_params.py:133]: I think I prefer your suggestion where we directly specify the argument
```suggestion
                   'If the auth mode is "login", provide the `--endpoint` or `--name` argument and your "az login" credentials will be used for authorization. ' +
```

Comment (ChristineWanjau) [reply src/azure-cli/azure/cli/command_modules/appconfig/_params.py]: Sure! This has been updated!

**Review by albertofori — APPROVED** [review]

**Comment by ChristineWanjau** [comment]
@yanzhudd or @zhoxing-ms this PR changes have been approved by @albertofori please help merge when you get some time.

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**